packages/runtime/runtime-definitions/src/test/types/validateRuntimeDefinitionsPrevious.generated.ts

```diff
@@ -642,6 +642,18 @@ declare function use_old_InterfaceDeclaration_IGarbageCollectionDetailsBase(
 use_old_InterfaceDeclaration_IGarbageCollectionDetailsBase(
     get_current_InterfaceDeclaration_IGarbageCollectionDetailsBase());
 
+/*
```

Comment:
I think you want to remove this file from PR. I'm also getting spurious validation changes here and there - really irritating, but not enough for me to try to dig into it. :)
@tylerbutler - anything we can do about it? I'm happy to share any info required to make it diagnosable.